### DISCUSSION TITLE
Recommended multipliers for different plant types

### DISCUSSION THREAD

**Discussion by ekutner** [jeroenterheerdt/HAsmartirrigation#512] [General]
Hi
I love the idea behind this integration and would like to start using it. I understand that the formula it is using is intended for short grass, so it would be perfect for a lawn, but I have dedicated irrigation lines for other types of plans as well, trees, medium size shrubs and smaller flowers. I know from experience that those other types of vegetation require significantly less watering than the lawn, but I was unable to find any recommended method for determining the right multiplier to put in the zone configuration. Is there any such known recommendation or should it be done just be trial and error?

Thank you

**Comment by jeroenterheerdt** [comment]
there is no such recommendation. I'd define multiple zones to see if you have figure out the right multipliers for your needs

**Comment by ekutner** [comment]
Iv'e defined a zone for every irrigation line I have but was hoping for something more "scientific", since obviously different plans need different levels of irrigation. I'll just use trial and error to figure out what multiplier to use. 
Thank you for a great integration.
